services/comfy/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.10.9-slim
+
+SHELL ["/bin/bash", "-ceuxo", "pipefail"]
+
+ENV DEBIAN_FRONTEND=noninteractive PIP_PREFER_BINARY=1
+
+RUN --mount=type=cache,target=/root/.cache/pip pip install torch==1.13.0 torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu117
+
+RUN apt update && apt install -y git && apt clean
```

Comment:
You may want to use apt-get instead of apt. apt is an user-facing UI for apt-get and it's siblings, that isn't mean for scripts. From apt's man page (`man apt`):

```
SCRIPT USAGE AND DIFFERENCES FROM OTHER APT TOOLS
       The apt(8) commandline is designed as an end-user tool and it may
       change behavior between versions. While it tries not to break backward
       compatibility this is not guaranteed either if a change seems
       beneficial for interactive use.

       All features of apt(8) are available in dedicated APT tools like apt-
       get(8) and apt-cache(8) as well.  apt(8) just changes the default value
       of some options (see apt.conf(5) and specifically the Binary scope). So
       you should prefer using these commands (potentially with some
       additional options enabled) in your scripts as they keep backward
       compatibility as much as possible. 
```

Reply:
I didn't know there was a difference in them. Since apt-get seems more suited for this, I switch back over to that. Thanks for that